src/components/SignatureModal/SignatureModal.js

```diff
@@ -100,6 +100,11 @@ class SignatureModal extends React.PureComponent {
     const { width, height } = canvas.getBoundingClientRect();
     canvas.width = width;
     canvas.height = height;
+
+    if (!this.signatureTool.isEmptySignature()) {
+      // if the canvas is resized the content get cleared. So redraw it
+      this.signatureTool.drawAnnot();
```

Comment:
Hmm `drawAnnot` is removed in 6.0 so I think we should go with the other way you suggested.

Also I think it'd be better if we have a separate function for this. Redrawing the canvas in a function called `setSignatureCanvasSize` is kind of weird imo.

Do you think it makes more sense to just clear the signature when an orientation happens? The signature that was drawn is out of position anyways because of the canvas size change?

Reply:
And I think for redraw we can use `ctx.getImageData` and `ctx.putImageData `

Reply:
Rotating and resizing isn't perfect but isn't too bad 

![signatureResize2](https://user-images.githubusercontent.com/45575633/69192505-8f810600-0ad9-11ea-8fc5-e1b75d64e190.gif)

Reply:
I refactor the code, I didn't end up with "getImageData" and "putImageData" because I wanted to do "drawImage(imageData, 0, 0, newWidth, newHeight)" for resizing the image